plugins/zns/zns.c

```diff
@@ -280,12 +269,17 @@ static int zns_mgmt_send(int argc, char **argv, struct command *cmd, struct plug
 		}
 	}
 
-	err = __zns_mgmt_send(fd, cfg.namespace_id, cfg.zslba,
-		cfg.select_all, cfg.timeout, zsa, 0, NULL);
-	if (!err)
+	err = nvme_zns_mgmt_send(fd, cfg.namespace_id, cfg.zslba, zsa,
+		cfg.select_all, 0, 0, NULL, cfg.timeout, &result);
+	if (!err) {
 		printf("%s: Success, action:%d zone:%"PRIx64" all:%d nsid:%d\n",
 			command, zsa, (uint64_t)cfg.zslba, (int)cfg.select_all,
 			cfg.namespace_id);
+		if(result && zsa == NVME_ZNS_ZSA_RESET) {
+			if(result & 0x1)
```

Comment:
It would be better to replace 0x1 with its constant mask.

E.g., Zone Capacity Changed Mask (ZCCM) or similar. Otherwise, it is hard to read.

Also, there's a missing space, and you might as well write it like this:

if (zsa == NVME_ZNS_ZSA_RESET && result & ZCCM) ...